### PR TITLE
Cherry-pick docs updates from dev-creatures

### PR DESCRIPTION
**Pull request by infiniterasadev**
Updates master to have the docs from dev-creatures. 
dev-creatures is a long way from being merged and this should be iterated on outside of that work.

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+Before you can build and run the code, there are some configuration steps for the database. 
+
+- Launch MySQL Workbench and select your `Local instance MySQL80` under `MySQL Connections` that was created during install
+- Enter your root password that you created to connect to it
+- In the Navigator panel on the left, right-click in the blank space and choose `Create Schema`
+  - In the new tab, name the first schema `rasaauth` and click `Apply`
+  - Repeat this two more times for `rasachar` and `rasaworld`
+- Double-click `rasaauth` in the `SCHEMAS` list which makes it active
+- Go to `File > Open SQL Script`
+- Select `Rasa.NET\database\full\auth_database.sql` file in the code directory that you clone in the above steps
```
Comment (Blumster) [inline docs/setup.md:77]: If we merge it to master, they won't find the sql files, as they are on another branch yet.

**Review by Blumster — CHANGES_REQUESTED** [review]

File: docs/setup.md (hunk 1 above)
Comment (infiniterasadev) [reply docs/setup.md:77]: True, thanks for bringing this up. I didn't find a good commit to pull over the correct files. 

Is it adequate enough for me to pull the creation scripts into master instead?

**Comment by infiniterasadev** [comment]
https://github.com/InfiniteRasa/Rasa.NET/pull/43/commits/23053428d44578a9d24f39b9f751672ace5f8e42 adds the `full` scripts.

**Comment by infiniterasadev** [comment]
@Blumster ready for another peek.

**Review by Blumster — APPROVED** [review]

**Review by Blumster — APPROVED** [review]

**Pull request merged**